pyoozie/client.py

```diff
@@ -79,11 +79,14 @@ def __init__(self, url=None, user=None, timeout=None, verbose=True, **_):
         self._verbose = verbose  # Note: change default for verbose!
         self._stats = OozieClient.Stats()
         self._valid_server = False
+        if not type(session) == requests.sessions.Session: 
+            session = requests.Session()
```

Comment:
In fact, is it even worth doing the typecheck? Do we just assume the caller knows what they're doing and only handle the `None` case?

Reply:
True. I thought it might be a nice way to make sure whatever is there is a session, but I guess the user will find out /really/ quickly if they aren't passing in a session

Reply:
🔥 'd it